.env

```diff
@@ -4,8 +4,8 @@ NETWORK_NODE_IMAGE_PREFIX=gcr.io/hedera-registry/
 NETWORK_NODE_IMAGE_NAME=main-network-node
 
 #### Image Tags/Hashes ####
-NETWORK_NODE_IMAGE_TAG=0.35.0-alpha.6
-HAVEGED_IMAGE_TAG=0.35.0-alpha.6
+NETWORK_NODE_IMAGE_TAG=0.36.0-alpha.1
```

Comment:
Is there a reason this tag is used which was created 2 weeks ago? whereas `0.35.1` was created a day ago and contains the update that prevents delegatecall to the precompile contract.

Reply:
Oh, forgot to commit my latest changes, it's fixed there. Also my test is failing. I though I created this PR as draft.